public/data/leabuende.json

```diff
@@ -0,0 +1,13 @@
+{	
+    "name": "Léa Buendé",	
```

Comment:
```suggestion
    "name": "Lea Buende",	
```
Can you change the letter `e` to simple form. As you will be easy to search on searchbar.

Reply:
Done, thank you @shyamtawli  !